lte/gateway/c/oai/tasks/nas/emm/sap/emm_cn.c

```diff
@@ -766,6 +766,7 @@ static int _emm_cn_activate_dedicated_bearer_req(
   esm_sap.data.eps_dedicated_bearer_context_activate.pco = msg->pco;
   // stole ref if any
   msg->pco = NULL;
+  esm_sap.data.eps_dedicated_bearer_context_activate.gtp_teid = msg->gtp_teid;
```

Comment:
gtp TE id is not needed in NAS

Reply:
Bearer context for the new dedicated bearer gets created at NAS similar to default bearer context. Hence gtp_teid is sent from MME to NAS. Once the bearer context is created, gtp_teid is stored in the context by NAS and MME fetches it from the context to send it in ERAB SETUP REQ to eNB

Reply:
As mentioned above, NAS shall not know anything about TE id. Please  change it , Lets discuss in next call.